test/driver/run_as_ppx_rewriter_preserve_version/print_magic_number.ml

```diff
@@ -1,5 +1,9 @@
 let magic_length = String.length Astlib.Config.ast_impl_magic_number
 let buf = Bytes.create magic_length
-let len = input stdin buf 0 magic_length
+
+let len =
+  set_binary_mode_in stdin true;
+  input stdin buf 0 magic_length
+
```

Comment:
Strictly speaking this is probably not needed because the magic number does not contain CRs. Also this test is not run on Windows :)

Reply:
reverted

Reply:
The way it is now, this test doesn't fail for you on Windows, @jonahbeckford , does it?

Reply:
Everything now passes on Windows using the latest ppxlib PR and two merlin PRs (one closed, one open).